lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -456,20 +480,36 @@ def delete_comment(request, course_id, comment_id):
     return JsonResponse(prepare_content(comment.to_dict(), course_key))
 
 
+def _vote_or_unvote(request, course_id, obj, value):
+    """
+    Vote or unvote for a thread or a response.
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
Since this has been uncovered, it should be fixed. This should apply to all techdebt - we enforce it with pep8/pylint, but we should all improve the codebase whenever we touch older areas. If you feel it's more appropriate to do so in a separate PR, then do that.